crates/sui-faucet/src/faucet/simple_faucet.rs

```diff
@@ -1312,70 +1364,24 @@ mod tests {
         }
     }
 
-    #[tokio::test]
-    async fn test_ttl_cache_expires_after_duration() {
```

Comment:
Why has this test disappeared?

Reply:
Thanks for catching this, just a mistake on my part.